mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -7,17 +7,45 @@ CLASS bqp net/minecraft/block/RedstoneWireBlock
 	FIELD f DIRECTION_TO_WIRE_CONNECTION_PROPERTY Ljava/util/Map;
 	FIELD g WIRE_CONNECTIONS_TO_SHAPE [Lcsp;
 	FIELD h wiresGivePower Z
+	FIELD i lateNeighborUpdates Ljava/util/Set;
 	METHOD <init> (Lbml$c;)V
 		ARG 1 settings
+	METHOD a increasePower (ILbvj;)I
+		ARG 1 power
+		ARG 2 state
+	METHOD a connectsTo (Lbgr;Lev;)Z
+		ARG 0 view
+		ARG 1 pos
 	METHOD a getRenderConnectionType (Lbgr;Lev;Lfa;)Lbwo;
 		ARG 1 view
 		ARG 2 pos
+		ARG 3 dir
+	METHOD a updateNeighborsIfWire (Lbhh;Lev;)V
```

Comment:
Sounds a bit silly to me

Reply:
This one updates the given blockpos's neighbors, and the neighbors of those neighbors, if the block in the given world at the given position is a `RedstoneWireBlock`

Reply:
`updateNeighboringWire`

Reply:
To me that sounds more like "look at the neighbors of this block, and update them if they are wire"

This method indiscriminately updates blocks, only the "root" has to be redstone wire

Reply:
Just updateNeighbors; the redstone wire updates its neighbors, the check is just to ensure it's not called after the redstone wire stops being a wire I guess?